library/src/include/util.hpp

```diff
@@ -42,18 +42,28 @@ namespace hiptensor
     }
 
     template <typename T>
-    static inline std::vector<T> stridesFromLengths(std::vector<T> const& lengths)
+    static inline std::vector<T> stridesFromLengths(std::vector<T> const& lengths, bool col_major = true)
```

Comment:
Does this interact with HIPTENSOR_DATA_LAYOUT_COL_MAJOR, as found in permutation_ck_impl.hpp?
@CongMa13

Reply:
yes, HIPTENSOR_DATA_LAYOUT_COL_MAJOR used in hiptensor_permutation to indicate the layout